job-service-contract/src/main/resources/com/hpe/caf/services/job/swagger.yaml

```diff
@@ -335,6 +337,55 @@ definitions:
         example:
           "tag:4": "4"
           owner: "bob"
+  expirationPolicy:
+    type: object
+    properties:
+      "Active":
+        $ref: "#/definitions/policy"
+      "Completed":
+        $ref: "#/definitions/policy"
+      "Failed":
+        $ref: "#/definitions/policy"
+      "Cancelled":
+        $ref: "#/definitions/policy"
+      "Waiting":
+        $ref: "#/definitions/policy"
+      "Paused":
+        $ref: "#/definitions/policy"
+      "Expired":
+        $ref: "#/definitions/deletePolicy"
+      "Default":
+        $ref: "#/definitions/policy"
+    description: The expiration policy to be applied on the job
+  deletePolicy:
+    type: object
+    properties:
+      expiryTime:
+        type: string
+        pattern: '^((lastUpdateDate)|(createDate))(\+P)([1-9]\d?|[12]\d{2}|3[0-5]\d|36[0-5])D$'
+        default: "createDate+10D"
+      expirationOperation:
+        type: string
```

Comment:
You created a separate object (definition) for this will the normal policy but you haven't done that here.  I think it would be better to have the symmetry of either doing it in both places or not doing it in either.

Reply:
extracted the object as expiryTimePattern